great_expectations/core/expectation_suite.py

```diff
@@ -548,13 +577,29 @@ def add_expectation(
                     found_expectation_indexes[0]
                 ] = expectation_configuration
             else:
+                if send_usage_event:
+                    usage_stats_event_name: str = "expectation_suite.add_expectation"
+                    usage_stats_event_payload: dict = {}
+                    self._data_context.send_usage_message(
+                        event=usage_stats_event_name,
+                        event_payload=usage_stats_event_payload,
+                        success=False,
+                    )
                 raise DataContextError(
                     "A matching ExpectationConfiguration already exists. If you would like to overwrite this "
                     "ExpectationConfiguration, set overwrite_existing=True"
                 )
         else:
             self.append_expectation(expectation_configuration)
 
+        if send_usage_event:
+            usage_stats_event_name: str = "expectation_suite.add_expectation"
+            usage_stats_event_payload: dict = {}
+            self._data_context.send_usage_message(
+                event=usage_stats_event_name,
+                event_payload=usage_stats_event_payload,
+                success=True,
+            )
```

Comment:
Helper method through `DataContext` needs to be used in this case because `send_usage_message()` in the `usage_statistics/util.py` method requires import of `ExpectationSuite` module,  which results in a circular import